src/antidote_crdt_set_rw.erl

```diff
@@ -86,67 +86,59 @@ new() ->
   {[], orddict:new()}.
 
 -spec value(set()) -> [member()].
-value({_,Dict}) ->
+value({_, Dict}) ->
     [Val || {Val, []} <- orddict:to_list(Dict)].
 
 
 -spec downstream(set_op(), set()) -> {ok, effect()}.
 downstream({add, Elem}, State) ->
     downstream({add_all, [Elem]}, State);
-downstream({add_all,Elems}, {ResetTokens, Dict}) ->
-    {ok, {add, elemsWithRemovedTombstones(lists:usort(Elems), Dict, ResetTokens), ResetTokens}};
+downstream({add_all, Elems}, {ResetTokens, Dict}) ->
+    {ok, {add, elems_with_removed_tombstones(lists:usort(Elems), Dict, ResetTokens), ResetTokens}};
 downstream({remove, Elem}, State) ->
     downstream({remove_all, [Elem]}, State);
 downstream({remove_all, Elems}, {ResetTokens, Dict}) ->
-    {ok, {remove, elemsWithRemovedTombstones(lists:usort(Elems), Dict, ResetTokens), unique(), ResetTokens}};
+    {ok, {remove, elems_with_removed_tombstones(lists:usort(Elems), Dict, ResetTokens), unique(), ResetTokens}};
 downstream({reset, {}}, {ResetTokens, Dict}) ->
-  RemovedTombstones = elemsWithRemovedTombstones(lists:usort(orddict:fetch_keys(Dict)), Dict, ResetTokens),
+  RemovedTombstones = elems_with_removed_tombstones(lists:usort(orddict:fetch_keys(Dict)), Dict, ResetTokens),
   {ok, {reset, RemovedTombstones, ResetTokens, unique()}}.
 
 %% @doc generate a unique identifier (best-effort).
 unique() ->
     crypto:strong_rand_bytes(20).
 
-elemsWithRemovedTombstones([], _Dict, _ResetTokens) -> [];
-elemsWithRemovedTombstones(Elems, [], ResetTokens) -> [{E,ResetTokens} || E <- Elems];
-elemsWithRemovedTombstones([E|ElemsRest]=Elems, [{K,Ts}|DictRest]=Dict, ResetTokens) ->
-    if
-        E == K ->
-            [{E,Ts++ResetTokens}|elemsWithRemovedTombstones(ElemsRest, DictRest, ResetTokens)];
-        E > K ->
-            elemsWithRemovedTombstones(Elems, DictRest, ResetTokens);
-        true ->
-            [{E,ResetTokens}|elemsWithRemovedTombstones(ElemsRest, Dict, ResetTokens)]
-    end.
-
+elems_with_removed_tombstones([], _Dict, _ResetTokens) -> [];
+elems_with_removed_tombstones(Elems, [], ResetTokens) -> [{E, ResetTokens} || E <- Elems];
+elems_with_removed_tombstones([E|ElemsRest], [{K, Ts}|DictRest], ResetTokens) when E == K ->
```

Comment:
I think the old version with the `if` is more readable. Here the important condition is lost among the pattern matches...